src/modules/github/components/connections/connections.tsx

```diff
@@ -2,19 +2,77 @@
 
 import { TrashIcon } from '@radix-ui/react-icons'
 import Link from 'next/link'
-import { type FC } from 'react'
+import { type FC, useState } from 'react'
 
+import { AlertDialog } from '@/elements/root/alert-dialog/alert-dialog'
 import { Avatar } from '@/elements/root/avatar/avatar'
 import { Button } from '@/elements/root/button/button'
 
+import { useDictionary } from '@/contexts/root/dictionary-provider/dictionary-provider'
+
 import { NewConnectionButton } from '@/components/github/root/new-connection-button/new-connection-button'
 
 import { useInstallations } from '@/hooks/github/root/use-installations'
 
 export const Connections: FC = () => {
+  const [dialogData, setDialogData] = useState<
+    | {
+        open: true
+        title: string
+        installationId: number
+        installationOwner: string
+      }
+    | {
+        open: false
+        title?: never
+        installationId?: never
+        installationOwner?: never
+      }
+  >({ open: false })
+  const { translate } = useDictionary()
   const { isLoading, installations, addInstallation, deleteInstallation } =
     useInstallations()
 
+  const handleActionClick = () => {
+    if (!dialogData.open) {
+      return
+    }
+    submit()
+  }
+
+  const submit = () => {
+    if (!dialogData.installationId) {
+      return
+    }
+    console.log('submit')
+    deleteInstallation(dialogData.installationId)
+      .then(() => {
+        console.log('success')
+        handleOpenChange(false)
+      })
+      .chatch(() => {
```

Comment:
chatch?

Reply:
lol I already modified.........................................I really didn't know that WHAT THE ERROR.